src/main.rs

```diff
@@ -218,6 +242,45 @@ async fn on_connect(socket: SocketRef, Data(auth): Data<AuthPayload>, pool: Stat
         },
     );
 
+    socket.on(
+        "play-again",
```

Comment:
We probably don't need this. If user clicks play again, on frontend, we can create `new State()` and then call `joinRoom()` with the previous room code. The board will automatically reset, and the socket id will be updated automatically on connection.

Reply:
ok